EasyPost.nuspec

```diff
@@ -11,6 +11,8 @@
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>EasyPost Shipping API Client Library for .NET https://easypost.com/docs</description>
         <tags>EasyPost ShippingAPI USPS UPS FedEx</tags>
+        <readme>docs\README.md</readme>
+        <icon>images\icon.png</icon>
```

Comment:
Question: are these paths? If so they don't line up with the locations of these.